app-former-showcase/app-former-webapp/src/main/resources/example-repositories.properties

```diff
@@ -13,4 +13,4 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-ls-playground=https://github.com/mbarkley/livespark-playground.git
+ls-playground=https://github.com/mbarkley/app-former-playground.git
```

Comment:
@ge0ffrey So you'd need to make sure any changes you make do not impact others' projects (e.g. you might need a "tag" for "experimental" or something too).